src/Data/Generic/Rep.purs

```diff
@@ -34,3 +37,6 @@ newtype Argument a = Argument a
 class Generic a rep | a -> rep where
   to :: rep -> a
   from :: a -> rep
+
+toRep :: forall from to. Generic from to => Proxy from -> Proxy to
```

Comment:
Also I don’t know about you but I frequently confuse `to` and `from`, and I’m sure having `to :: rep -> a` but `toRep :: Proxy a -> Proxy rep` is going to be even more confusing to me 😬 May I suggest `repOf` instead?

Reply:
Sounds good to me!